nifi-commons/nifi-properties/src/test/java/org/apache/nifi/util/StringUtilsTest.java

```diff
@@ -0,0 +1,84 @@
+package org.apache.nifi.util;
```

Comment:
Needs the Apache License header, see the other file you worked on (TestFormatUtils.java) for an example (but almost any file in the codebase should have it)

Reply:
Added the license header.